__tests__/lint/assertions-defined-error/.redocly.yaml

```
@@ -4,16 +4,17 @@ apis:
 
 lint:
   rules:
-    assertions:
-      - context:
-        - type: Response
-          excludeParentKeys: ['200']
-        subject: Response
-        property: content
-        message: Response content should be defined
-        defined: true
-      - subject: Operation
-        property: security
-        message: Operation security should be defined
-        defined: true
+    assert/response-content-defined:
+      context:
+      - type: Response
+        excludeParentKeys: ['200']
```

```suggestion
        excludeParentKeys: ['204']
```
A practical use case is to adjust this one for 204 (HTTP code means no content). I realize you didn't write these test cases, but this improves them for real-world use-cases. This will allow future developers to contribute more easily, and prospective users to review the tests which will have better semantic meaning.

---

It makes perfect sense, thanks :)